### DISCUSSION TITLE
Certificate Validation

### DISCUSSION THREAD

**Discussion by uvarvu** [usebruno/bruno#798] [Q&A]
Is there a way to turn this off?   I'm currently unable to send any requests because SSL is enabled.

**Comment by helloanoop** [comment]
Yes, please click on the gear icon in the bottom left corner. This will open a preferences modal where you can turn ssl validation off.

**Reply by richemmanu** [reply]
I still have this challenge even after disabling the ssl checkbox is it a version issue am on version: v1.38.1
![image](https://github.com/user-attachments/assets/3fa4a85e-9a7d-477d-89f9-8fceb6f784b8)

**Reply by jonh001** [reply]
Is this SSL/TLS thing still an issue?
It doesn't seem to do anything.  I'm still getting **self signed certificate in certificate chain**
It says: You can disable SSL verification in the Preferences. To open the Preferences, click on the gear icon in the bottom left corner..... but this does nothing. 
Same error even after including the CA certificate
